solr/core/src/java/org/apache/solr/cli/ApiTool.java

```diff
@@ -57,27 +57,28 @@ public List<Option> getOptions() {
             .hasArg()
             .required(true)
             .desc("Send a GET request to a Solr API endpoint.")
-            .build());
+            .build(),
+        SolrCLI.OPTION_CREDENTIALS);
   }
 
   @Override
   public void runImpl(CommandLine cli) throws Exception {
     String response = null;
     String getUrl = cli.getOptionValue("get");
     if (getUrl != null) {
-      response = callGet(getUrl);
+      response = callGet(getUrl, cli.getOptionValue("credentials"));
```

Comment:
wondering if instead of hardcoding the name to `"credentials"` everywhere we could actually use `cli.getOptionValue(SolrCLI.OPTION_CREDENTIALS.getOpt())` instead.

Reply:
i like that...   It is probably something we should do everywhere.   I always struggle with "forbiddenapis" stuff, but you could make a case that "cli.getOptionValue(String)" shouldn't be permitted...